### DISCUSSION TITLE
"Cannot copy different files to:" warning but cant understand why

### DISCUSSION THREAD

**Discussion by AlexMC** [emmercm/igir#877] [Q&A]
I have the complete FBNeo Neo Geo set, and I'm trying to create a 1G1R set for JELOS.

This is my script:
```
npx igir@2.1.1 copy test clean \
  --dat "./FinalBurn*.zip" \
  --input "./FinalBurn/Neo Geo/" \
  --output "${DESTINATION}/{jelos}" \
  --overwrite-invalid \
  --no-bad \
  --single \
  --prefer-parent
```

When I run it I get this for every single game:
```
WARN:  Cannot copy different files to: jelos-test/neogeo/19yyo.zip:
WARN:    FinalBurn/Neo Geo/19yy.zip
WARN:    FinalBurn/Neo Geo/19yyo.zip
WARN:  Cannot copy different files to: jelos-test/neogeo/2020bba.zip:
WARN:    FinalBurn/Neo Geo/2020bba.zip
WARN:    FinalBurn/Neo Geo/2020bb.zip
WARN:    FinalBurn/Neo Geo/19yy.zip
WARN:  Cannot copy different files to: jelos-test/neogeo/2020bbh.zip:
WARN:    FinalBurn/Neo Geo/2020bbh.zip
WARN:    FinalBurn/Neo Geo/2020bb.zip
WARN:    FinalBurn/Neo Geo/19yy.zip
WARN:  Cannot copy different files to: jelos-test/neogeo/3countba.zip:
WARN:    FinalBurn/Neo Geo/3countba.zip
WARN:    FinalBurn/Neo Geo/3countb.zip
WARN:    FinalBurn/Neo Geo/19yy.zip
...
...
...
```

I'm not getting why it happens in the first place, but even less why it always shows the 19yy.zip for each one...

Does anyone get it?

**Comment by emmercm** [comment]
This might be related to https://github.com/emmercm/igir/discussions/857. So that I can try to reproduce, where did you download that DAT zip from?

**Reply by AlexMC** [reply]
The dat was downloaded from here: https://github.com/libretro/FBNeo/tree/master/dats (The NeoGeo one)
I also tried to rerun it using the PR you suggested in #857 and got the exact same results.

**Reply by emmercm** [reply]
I wasn't able to reproduce the issue. Taking one example such as 19yyo, can you paste the contents of the input files? This can be done from the CLI with a command such as:

```shell
unzip -v "FinalBurn/Neo Geo/19yy.zip"
```

**Comment by emmercm** [comment]
To answer the original question...

> I'm not getting why it happens in the first place, but even less why it always shows the 19yy.zip for each one...

the log line can happen when "raw copying" (not extracting or zipping) and there isn't one input archive that contains every necessary file. I'll see if I can make the log line a little more explicit.

The issue will go away if you use the `igir zip` command to recreate the zip files when writing.
